### DISCUSSION TITLE
Thumbnail format

### DISCUSSION THREAD

**Discussion by aledeg** [getnikola/nikola#3549] [Q&A] [answered]
Is there a way to choose the output format for thumbnails? I am not talking about the size (I've found the option) but about the file format (PNG vs JPG vs Progressive JPG).
For accessibility purpose, I'd like to have those in Progressive JPG instead of baseline JPG.

**Comment by Kwpolska** [comment, marked as answer]
No. This could be added quite easily (add `progressive=True` to the Pillow save arguments [here](https://github.com/getnikola/nikola/blob/master/nikola/image_processing.py#L149) if output format is JPEG) — but is it really worth it for thumbnails? Does it have any negative effect on the output file size?

If there are no drawbacks, we can take a PR making this change.

**Reply by aledeg** [reply]
Thank you!
I have quite large thumbnails. So yes.
But it's quite a limited use-case.

I won't commit on any PR since my python knowledge is quite limited :)

**Reply by aledeg** [reply]
I think I've found a way. I used a filter with **jpegoptim**. In my *conf.py* file, I've added the following:
```python
FILTERS = {
    ".jpg": ["jpegoptim --strip-all --all-progressive %s"],
}
```

That works for what I need. It might help others!
